src/gap.c

```diff
@@ -1374,75 +1341,91 @@ Obj FuncSetUserHasQuit( Obj Self, Obj value)
    
  Obj FuncCALL_WITH_TIMEOUT( Obj self, Obj seconds, Obj microseconds, Obj func, Obj args )
 {
-  Obj plain_args;
     Obj res;
     Obj currLVars;
     Obj result;
     Int recursionDepth;
     Stat currStat;
+    UInt newJumpBuf = 1;
+    UInt mayNeedToRestore = 0;
+    Int iseconds, imicroseconds;
+    Int curr_seconds, curr_microseconds, curr_nanoseconds;
+    Int restore_seconds = 0, restore_microseconds = 0;
+      
     if (!SyHaveAlarms)
       ErrorMayQuit("CALL_WITH_TIMEOUT: timeouts not supported on this system", 0L, 0L);
     if (!IS_INTOBJ(seconds) || 0 > INT_INTOBJ(seconds))
-      ErrorMayQuit("CALL_WITH_TIMEOUT(<seconds>, <microseconds>, <func>,<args>): <seconds> must be a non-negative small integer",0,0);
+      ErrorMayQuit("CALL_WITH_TIMEOUT(<seconds>, <microseconds>, <func>,<args>):"
+		   " <seconds> must be a non-negative small integer",0,0);
```

Comment:
This seems like a purely cosmetical change; well, if you do that already, perhaps insert a space in `<func>,<args>`?